zoneminder/zoneminder-event-cleanup.sh

```diff
@@ -4,23 +4,31 @@
 # Purpose:	Clean up old Zoneminder events but exclude archived events
 # Authors:	Claudio Kuenzler (2018)
 # Doc:		https://www.claudiokuenzler.com/blog/814/how-to-manually-clean-up-delete-zoneminder-events
-# History:	
-# 2018-12-14 First version
+# History:
+# 2020-08-06 changed for zoneminder 1.24+ (@Brawn1)
+# 2020-07-24 added Docker Mysql (@Brawn1)
 # 2018-12-18 DB Name variable in first query
+# 2018-12-14 First version
 ###########################################################################
 # User variables
-olderthan=2 # Defines the minimum age in days of the events to be deleted
-zmcache=/var/cache/zoneminder/events # Defines the path where zm stores events
+olderthan=7 # Defines the minimum age in days of the events to be deleted
+zmcache=/var/lib/docker/volumes/zoneminder_zmcache/_data/events # Defines the path where zm stores events
```

Comment:
The hard-coded default path should stay `/var/cache/zoneminder/events`, I honestly don't think there are more production Docker installations of ZM than real machines out there. Another parameter for easily defining another zm events path would be good here.

Reply:
ja sorry, i see that after the Pull Request.
i change that today with the optional feature. :-)